main.go

```diff
@@ -49,6 +39,11 @@ func main() {
 					Value: "",
 					Usage: "Interface to listen on, default is all",
 				},
+				cli.StringFlag{
+					Name:  "dbpath",
+					Value: "",
+					Usage: "Path to the database, default is current directory.",
```

Comment:
@ajvb What kind of database? If it's just boltdb, than perhaps make the cmd line flag more explicit to allow for further expansion without harming backwards compatibility.